menas/ui/service/SchemaService.js

```diff
@@ -15,66 +15,77 @@
 
 jQuery.sap.require("sap.m.MessageBox");
 var SchemaService = new function() {
-
-  var model = sap.ui.getCore().getModel();
-
-  this.getSchemaList = function(bLoadFirst, bGetAllVersionsOfFirst) {
-    Functions.ajax("api/schema/list", "GET", {}, function(oData) {
-      model.setProperty("/schemas", oData)
-      if(oData.length > 0 && bLoadFirst) {
-        SchemaService.getSchemaVersion(oData[0]._id, oData[0].latestVersion)
-      } else if(bGetAllVersionsOfFirst) {
-        SchemaService.getAllSchemaVersions(oData[0]._id)
-      }
-    }, function() {
-      sap.m.MessageBox.error("Failed to get the list of schemas. Please wait a moment and try reloading the application")
-    })
-  };
-
-  this.getLatestSchemaVersion = function(sId) {
-    Functions.ajax("api/schema/detail/" + encodeURI(sId) + "/latest", "GET", {}, function(oData) {
-      model.setProperty("/currentSchema", oData)
-      SchemaService.getSchemaUsedIn(oData.name, oData.version)
-    }, function() {
-      sap.m.MessageBox.error("Failed to get the detail of the schema. Please wait a moment and try reloading the application")
-      window.location.hash = "#/schema"
-    })
-  };
-
-  this.getSchemaVersion = function(sId, iVersion, sModelPath) {
-    var modelPath;
-    if(sModelPath) modelPath = sModelPath
-    else modelPath = "/currentSchema"
-    Functions.ajax("api/schema/detail/" + encodeURI(sId) + "/" + encodeURI(iVersion), "GET", {}, function(oData) {
-      model.setProperty(modelPath, oData)
-      SchemaService.getSchemaUsedIn(oData.name, oData.version)
-    }, function() {
-      sap.m.MessageBox.error("Failed to get the detail of the schema. Please wait a moment and try reloading the application")
-      window.location.hash = "#/schema"
-    })
-  };
-
-  this.updateSchema = function(sId, iVersion, sDesc) {
-    Functions.ajax("api/schema/edit", "POST", {
-      name: sId,
-      version: iVersion,
-      description: sDesc
-    }, function(oData) {
-      model.setProperty("/currentSchema", oData)
-      SchemaService.getSchemaList();
+	
```

Comment:
What's up with the indentation changes? We settled on 2-spaces in `.js`. 
Maybe check for a `.editorconfig` plugin for eclipse?

Reply:
Yeah doesn't support js, but this is me not doing autoformat, got it configured accordingly.